app/views/users/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<section class="main">
+  <h1 class="title">BudgetOnLine</h1>
+  <div class="btns">
+    <button class="login" type="button">
+      <%= link_to "SIGN IN", new_user_session_path %>
+    </button>
+    <button class="signup" type="button">
+      <%= link_to "SIGN UP", new_user_registration_path %>
+    </button>
```

Comment:
- Kindly update your splash screen to follow the design guidelines and follow the guidelines:
   - [ ] The App name has more spacing from the top
   - [ ] The login and signup buttons are at the bottom of the page.
   - [ ] The signup button needs more top and bottom spacing.

| Your design | Guidelines |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/230420807-5907b8c5-d52e-4aeb-898d-755da2199dc4.png) | ![image](https://user-images.githubusercontent.com/24703171/219884305-f1437aeb-2f46-4261-81df-9ff3aaa904dd.png) |

Reply:
Please Kindly Implement this requested change from the previous code reviewer

Reply:
Your sign up and sign in buttons should match those of given design so it would be nice if you changed this.